File: WebApp/src/app/core/http/dto/menu-dto/menu-item/create-menu-item.dto.ts
export interface CreateMenuItemDto {
  menuType: string; // "drinks" or "food"
  name: string;
  description: string;
  imageUrl?: string | null;
  price: number;

  nutritionalValues: string;
  ingredients: string;
  allergens: string;

  position: number;
  menuCategoryId: number;
  menuId: number;
}

File: WebAPI/WebAPI.Application/DTO/MenuDTO/MenuItem/CreateMenuItemDto.cs
namespace WebAPI.Application.DTO.MenuDTO.MenuItem;

public class CreateMenuItemDto
{
    public string MenuType { get; set; } // "drinks" or "food"
    public string Name { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
    public string? ImageUrl { get; set; } = null;
    public double Price { get; set; }
 
    
    public string NutritionalValues { get; set; } = string.Empty;
    public string Ingredients { get; set; } = string.Empty;
    public string Allergens { get; set; } = string.Empty;
    
    public int Position { get; set; }
    public int MenuCategoryId { get; set; }
    public int MenuId { get; set; }
}